### ISSUE TITLE
Tab+Ctrl+Esc keys are not passed to interactive ssh sessions

### ISSUE THREAD

**Issue by rafal-1990** [holzschu/a-shell#35]
Pressing the TAB-, CTRL- or ESC-button is interpreted locally by a-shell instead of passing to an interactive ssh session. E. g. if I press the TAB button I expect command completion in the ssh session instead of inserting a tab character. Or pressing Ctrl+C should abort the running command in the ssh session instead of exiting the session. And furthermore it is impossible to use vi within an interactive ssh session. But there seem to be other reasons for the remote vi(m) issue than that the key is not passed to the ssh session. Feel free to split this issue into two parts (one for „keys are not passed in ssh session“ and one for „vi(m) is not working in ssh session“)

**Comment by holzschu** [comment]
I'll have a look. There is something odd, because normally all input is already forwarded to ssh sessions (i.e. I wrote code that should do this; it seems it is not triggered, so I need to understand why).
As for vim not starting, do you have an error message?

**Comment by holzschu** [comment]
I checked on my side, and tab, ctrl, esc are passed to the ssh session on my iPad. I can also start vim (although it does not have the right window size).
Which command are you using to start ssh?

**Comment by rafal-1990** [comment]
Yes, there was really something odd ;) I am using „ssh user@192.168.1.12“ (and I am wondering why the `known_hosts `file isn‘t created in .ssh dir but in the `Documents` dir but this is actually another issue).
I‘ve just noted that you released a new version of a-shell and after I’ve updated the app to the new version, the key forwarding works as expected. Now I‘am also able to start vim within the ssh session. Before the update there where only displayed some ANSI escape sequences.
